ui/src/app/login/controllers/LoginCtrl.js

```diff
@@ -3,19 +3,19 @@
   angular
     .module('app')
     .controller('LoginCtrl', [
-      '$scope', '$state', 'Security',
+      '$scope', '$state', 'UserService',
       LoginCtrl
     ]);
 
-  function LoginCtrl($scope, $state, Security) {
+  function LoginCtrl($scope, $state, UserService) {
     console.log("In control");
     $scope.user = {};
     $scope.incorrectCredentials = false;
 
-    Security.removeToken();
+    UserService.removeToken();
```

Comment:
Have this after all the declarations and initialization. (After $scope.login = .....)

Also noticed there are couple of console logs in this file. Remove them if they are not required. Or have them in the debug mode.